src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -118,6 +124,20 @@ public Integer convert(String value) {
         }
     }
 
+    @Option(names = "--depth", description = "Perform shallow validation in which each SHACL rule (shape) is only applied to [num] target nodes, or 50 nodes if [num] is omitted (requires -t).",
+            paramLabel = "num", arity = "0..1", converter = DepthConverter.class)
+    private int depth = DEFAULT_DEPTH;
+    private static class DepthConverter implements CommandLine.ITypeConverter<Integer> {
+        @Override
+        public Integer convert(String value) {
+            try {
+                return "".equals(value) ? MINIMUM_DEPTH : Integer.parseInt(value);
```

Comment:
I believe this is where you intend to set the value to the default if num was omitted. The descriptions say the is 50,  but MINIMUM_DEPTH is set to 1.